tools/file_packager.py

```diff
@@ -57,6 +57,7 @@
     subdir\file, in JS it will be subdir/file. For simplicity we treat the web platform as a *NIX.
 """
 
+import base64
```

Comment:
Nice that python makes this easy...

Reply:
Yeah is was rather easy to encode the files. Adding decoding was a little trickier but not by much, it's a small change with a visible impact.